### ISSUE TITLE
Problem with protob when using the lastest firmware version

### ISSUE THREAD

**Issue by Senyoret1** [skycoin/hardware-wallet-daemon#70]
**Describe the bug**
When using the lastest firmware version, I’m getting the following error when calling some endpoints:
```
{
    "error": {
        "message": "proto: wrong wireType = 0 for field Message",
        "code": 401
    }
}
```

**Environment information:**

- OS: Windows 10
- Platform: X64
- Firmware Version: commit 46ef0d3

**Steps to Reproduce**
Call the `/cancel` or `/wipe` API endpoints and you will get an error response. It also happens with some other endpoints.

**Actual behavior**
The response, no matter what action the user performs, is:
```
{
    "error": {
        "message": "proto: wrong wireType = 0 for field Message",
        "code": 401
    }
}
```

**Expected behavior**
The response should correspond to the result of the operation.

**Additional context**

**Possible implementation**

**Comment by therealssj** [comment]
should be fixed in #72 
please reopen if still the case

**Comment by Senyoret1** [comment]
Working well
